tools/db_crashtest.py

```diff
@@ -846,14 +846,8 @@ def exit_if_stderr_has_errors(stderr, print_stderr=True):
 
     stderrdata = stderr.lower()
     errorcount = stderrdata.count("error") - stderrdata.count("got errors 0 times")
-    print("#times error occurred in output is " + str(errorcount) + "\n")
+    print("#times std::error occurred in output is " + str(errorcount) + "\n")
```

Comment:
Should we print out `stderr` if error count > 0? Otherwise when we see this log we won't know what was printed.

Also what is std::error?

Reply:
stderr is being printed out in line 845. It's not visible in this diff. Let me remove std::error from the line.

Reply:
I'm only removing exit on stderr. It'll still print the errors for debugging or to address missing cases in case crash test exits successfully with valid stderr

Reply:
You are right..I missed the print there.